plugins/importexport/native/NativeImportExportPlugin.inc.php

```diff
@@ -353,14 +355,10 @@ function executeCLI($scriptName, &$args) {
 
 		switch ($command) {
 			case 'import':
-				$userName = array_shift($args);
-				$user = $userDao->getByUsername($userName);
+				if (!isset($user)) {
+					echo __('plugins.importexport.common.cliError') . "\n";
+					echo __('plugins.importexport.native.error.unknownUser', array('userName' => $userName)) . "\n\n";
 
-				if (!$user) {
-					if ($userName != '') {
-						echo __('plugins.importexport.common.cliError') . "\n";
-						echo __('plugins.importexport.native.error.unknownUser', array('userName' => $userName)) . "\n\n";
-					}
 					$this->usage($scriptName);
```

Comment:
Update the `plugins.importexport.native.cliUsage` locale key to include the `--user_name` parameter.

Reply:
Fixed

Reply:
Did you maybe forget to include this in the commit?

Reply:
it seems so. I have just committed the change